### PR TITLE
VZ:6938: Update vz to use Weblogic Operator 3.4.3

### PR DESCRIPTION
**Pull request by bobdonat**
This pull request updates vz to use Weblogic Operator 3.4.3.

### CODE REVIEW

**Comment by michael-cico** [comment]
No CRD changes ?

**Comment by bobdonat** [comment]
> No CRD changes ?

2 bug fixes only with no CRD changes.

File: platform-operator/thirdparty/charts/weblogic-operator/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # serviceAccount specifies the name of the ServiceAccount in the operator's namespace that the
 # operator will use to make requests to the Kubernetes API server.
 # The customer is responsible for creating the ServiceAccount in the same namespace as this Helm release.
-# If not specified, the operator will use the Helm release namespace's 'default' ServiceAccount.
+# If not specified, the the operator will use the Helm release namespace's 'default' ServiceAccount.
```
Comment (stevshap) [inline platform-operator/thirdparty/charts/weblogic-operator/values.yaml:7]: typo? two "the"s

**Review by stevshap — APPROVED** [review]

File: platform-operator/thirdparty/charts/weblogic-operator/values.yaml (hunk 1 above)
Comment (bobdonat) [reply platform-operator/thirdparty/charts/weblogic-operator/values.yaml:7]: that typo existed in their updated values.yaml.  I can fix that

Comment (bobdonat) [reply platform-operator/thirdparty/charts/weblogic-operator/values.yaml]: fixed

**Review by stevshap — APPROVED** [review]

**Pull request merged**